packages/bootstrap/scss/grid/_variables.scss

```diff
@@ -339,6 +339,42 @@ $kendo-grid-row-resizer-active-bg: k-color(primary) !default;
 $kendo-grid-row-resizer-height: k-spacing(0.5) !default;
 
 
+/// The horizontal padding of the Grid stack layout cells.
+/// @group grid
+$kendo-grid-stack-cell-padding-x: k-spacing(2) !default;
+/// The vertical padding of the Grid stack layout cells.
+/// @group grid
+$kendo-grid-stack-cell-padding-y: k-spacing(1.5) !default;
+
+/// The column gap of the Grid stack layout.
+/// @group grid
+$kendo-grid-stack-column-gap: k-spacing(1) !default;
+/// The column gap of the editable Grid stack layout.
+/// @group grid
+$kendo-grid-stack-editable-column-gap: k-spacing(2) !default;
+
+/// The width of the hierarchy column in the Grid stack layout.
+/// @group grid
+$kendo-grid-stack-hierarchy-col-width: k-spacing(2) !default;
+
+/// The font size of the Grid stack layout column header.
+/// @group grid
+$kendo-grid-stack-header-font-size: var( --kendo-font-size-sm, inherit ) !default;
+/// The font weight of the Grid stack layout column header.
+/// @group grid
+$kendo-grid-stack-header-font-weight: var( --kendo-font-weight-bold, inherit ) !default;
```

Comment:
I wonder in case `--kendo-font-weight-bold` is not defined, should the default value be `inherit` or rather `normal`

> Note: Applies to all themes

Reply:
I've changed it to bold